Forms/ConfigPanels/UCNames.cs

```diff
@@ -111,37 +147,24 @@ private void UpdateTextBoxColor()
             }
         }
 
-        public Genie.Names NameList
-        {
-            get
-            {
-                return m_NameList;
+        private void UCWindows_Load(object sender, EventArgs e)
+        {
+            try
+            {
+                PopulateList();
             }
-
-            set
-            {
-                m_NameList = value;
+            catch (NullReferenceException)
+            {
+                // todo

```

Comment:
this should probably be handled somehow or a comment explaining why it's not. It _is_ marked todo

Reply:
did try because an error was being thrown, it stopped it and not sure what to do next or stop it to begin with.